app/views/content/train-to-be-a-teacher/teaching-degree-apprenticeships.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Teacher degree apprenticeships (TDAs)"
+heading: "Teacher degree apprenticeships (TDAs)"
+description: |-
+ A new route to become a teacher without a degree, a teacher degree apprenticeship (TDA) lets you work at a school while you qualify as a teacher.
+related_content:
+    Is teaching right for me? : "/is-teaching-right-for-me"
+    Train to teach if you do not have a degree : "/train-to-be-a-teacher/if-you-dont-have-a-degree"
+    Who do you want to teach? : "/is-teaching-right-for-me/who-do-you-want-to-teach"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-undergraduate-course
+navigation: 20.20
+navigation_title: Teacher degree apprenticeships (TDAs)
```

Comment:
I don't think this works alongside the other three cards, all of which start `If you.....`

It would be better as `If you want to do an apprenticeship` perhaps?